lib/block/transaction.go

```diff
@@ -137,22 +137,7 @@ func (bt *BlockTransaction) Save(st *storage.LevelDBBackend) (err error) {
 	if err = st.New(bt.NewBlockTransactionKeyByBlock(bt.Block), bt.Hash); err != nil {
 		return
 	}
-	for _, op := range bt.transaction.B.Operations {
-		var bo BlockOperation
-		bo, err = NewBlockOperationFromOperation(op, bt.transaction, bt.blockHeight)
-		if err != nil {
-			return
-		}
-		if err = bo.Save(st); err != nil {
-			return
-		}
-		if pop, ok := op.B.(operation.Payable); ok {
-			target := pop.TargetAddress()
-			if err = st.New(bt.NewBlockTransactionKeyByAccount(target), bt.Hash); err != nil {
-				return
-			}
-		}
-	}
+
 	event := "saved"
 	event += " " + fmt.Sprintf("source-%s", bt.Source)
 	event += " " + fmt.Sprintf("hash-%s", bt.Hash)
```

Comment:
Since we will now trigger the event before everything is saved, what's the strategy to deal with race condition ?

Reply:
I think that should be handled right now. Safety trumps any performance concern.

Reply:
@Geod24 This problem is not derived from this PR, it already exists after `Batch`. It would be better with another issue.